README_zh.md

```diff
@@ -0,0 +1,43 @@
+# Kubebb Core
+
+[![codecov](https://codecov.io/gh/kubebb/core/branch/main/graph/badge.svg?token=TBPAVEZV2K)](https://codecov.io/gh/kubebb/core)
+
+[English](./README.md) | 简体中文
+
+Kubebb Core 提供了组件生命周期管理的核心实现。我们的设计和开发遵循 [operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)，实现Kubernetes API的扩展。
+
+![arch](./assets/arch.png)
+
+## 为什么选择 Kubebb-Core？
+
+- 声明式的组件生命周期管理，完全兼容 Helm 生态系统
+- 与低代码平台结合，为 Kubernetes 应用程序开发和部署提供全栈解决方案
+- 使用订阅自动升级组件
+- 灵活和强大的覆盖机制
+
+## 文档
+
+要了解有关 KubeBB Core 的更多信息，请[查看完整文档](https://kubebb.github.io/website/)。
+
+要快速开始使用 KubeBB Core，请[查看快速入门](https://kubebb.github.io/website/docs/category/快速开始)。
+
+## 贡献
+
+如果您想为 Kubb Core 做出贡献，请参阅[贡献指南](CONTRIBUTING.md)。
```

Comment:
Should these documents also be put on the website?
Make the code base no longer record changes to these documents

Reply:
Seems we still need such a document under the root directory event it is empty,so people konw where to find the real one.